tests/test_edges.cpp

```diff
@@ -116,6 +118,61 @@ TEST_CASE("EdgePopulation", "[edges]")
 }
 
 
+namespace {
+
+// TODO: remove after switching to C++17
+void copyFile(const std::string& srcFilePath, const std::string& dstFilePath)
+{
+    std::ifstream src(srcFilePath, std::ios::binary);
+    std::ofstream dst(dstFilePath, std::ios::binary);
+    dst << src.rdbuf();
+}
+
+}  // unnamed namespace
+
+
+TEST_CASE("EdgePopulation::writeIndices", "[edges]")
+{
+    const std::string srcFilePath = "./data/edges-no-index.h5";
+    const std::string dstFilePath = "./data/edges-no-index.h5.tmp";
```

Comment:
another shortcut: no "proper" temporary file (once again, postponed until `std::filesystem` is available)

Reply:
can you add the same TODO wrt c++17 about std::filesystem?